zone/inventory.cpp

```diff
@@ -814,8 +814,8 @@ void Client::DropItem(int16 slot_id, bool recurse)
 	LogInventory("[{}] (char_id: [{}]) Attempting to drop item from slot [{}] on the ground",
 		GetCleanName(), CharacterID(), slot_id);
 
-	if(GetInv().CheckNoDrop(slot_id, recurse) && RuleI(World, FVNoDropFlag) == 0 ||
-		RuleI(Character, MinStatusForNoDropExemptions) < Admin() && RuleI(World, FVNoDropFlag) == 2)
+	if(GetInv().CheckNoDrop(slot_id, recurse) && (RuleI(World, FVNoDropFlag) == 0 ||
+		RuleI(Character, MinStatusForNoDropExemptions) < Admin() && RuleI(World, FVNoDropFlag) == 2))
```

Comment:
Logic looks off here.
`nodrop && == 0 || < Admin && == 2`
versus
`nodrop && (== 0 || (< Admin && == 2))`

Reply:
Technically && is evaluated before || and almost no one ever remembers this (I had to look it back up to double-check). However, I was intending to fix that ambiguity in this PR and missed it here. I agree that adding the parenthesis is better and will update the PR for both comments you made.